src/transformers/modeling_utils.py

```diff
@@ -1927,7 +1927,7 @@ def cuda(self, *args, **kwargs):
     @wraps(torch.nn.Module.to)
     def to(self, *args, **kwargs):
         # Checks if the model has been loaded in 8-bit
-        if getattr(self, "is_quantized", False):
+        if getattr(self, "quantization_method", None) == QuantizationMethod.BITS_AND_BYTES:
```

Comment:
Are all of those supported with GPTQ?

Reply:
Yes, we can move the model without issue.